docs/products/alloydbomni/reference/list-of-extensions.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Extensions for Aiven for AlloyDB Omni
+sidebar_label: Extensions
+---
+
+Aiven for AlloyDB Omni extensions enhance capabilities of your service by adding advanced
+functionalities such as performance optimizations or additional data types.
+
+## List supported extensions
+
+To list the extensions and their details, such as extension version numbers,
+connect to your service and run:
+
+```sql
+SELECT *
+FROM pg_available_extensions;
+```
+
+## Limitations
+
+- Some extensions have dependencies and need to be created in a predetermined order.
+- Some extensions require resetting the client connection before they are fully available.
+- Most of superuser-only and untrusted extensions listed in `pg_available_extensions`
+  cannot be installed.
+
+  - To list superuser-only and untrusted extensions **available for installation**, run
+
+    ```sql
+    SELECT *
+    FROM pg_available_extension_versions
+    WHERE name = ANY(string_to_array(current_setting('extwlist.extensions'), ','));
+    ```
+
+  - To list all superuser-only and untrusted extensions, run
+
+    ```sql
+    SELECT name, version
+    FROM pg_available_extension_versions
+    WHERE superuser AND NOT trusted
+    ```
+
+## List of extensions
+
+<!-- vale off -->
+
+### Data types
+
+-   [citext](https://www.postgresql.org/docs/current/citext.html): Data
+    type for case-insensitive character strings
+-   [cube](https://www.postgresql.org/docs/current/cube.html): Data type
+    for multidimensional cubes
+-   [hstore](https://www.postgresql.org/docs/current/hstore.html): Data
+    type for storing sets of (key, value) pairs
+-   [ip4r](https://github.com/RhodiumToad/ip4r): Provides data types for IPv4/v6 addresses,
+    and address ranges, plus index support
+-   [isn](https://www.postgresql.org/docs/current/isn.html): Data types
+    for international product numbering standards
+-   [ltree](https://www.postgresql.org/docs/current/ltree.html): Data
+    type for hierarchical tree-like structures
+-   [pgvector](https://github.com/pgvector/pgvector): Type for vector
+    similarity search; `PG13 and newer`
+-   [postgresql-hll](https://github.com/citusdata/postgresql-hll): Type for storing
+    `hyperloglog` data; `PG11 and newer`
```

Comment:
AlloyDB is currently 15 only, and will therefore never have a version inferior to 15 so I would remove all references to the versioning number.